spec/javascript/components/supplemental_application/__snapshots__/SupplementalApplicationPage.test.js.snap

```diff
@@ -7131,9 +7241,15 @@ exports[`SupplementalApplicationPage should render the status update dropdown bu
                                                     "is_lottery_complete": false,
                                                     "lease": undefined,
                                                     "listing": Object {
+                                                      "account": Object {},
```

Comment:
hm. this looks like these fields were not being included at all previously, but now they're being included as empty objects. Could that potentially have other ramifications?

Reply:
@akegan really great question -- I looked and yes it has some side effects e.g. it renders blank fields on forms if there is no value for adding preferences on the add application page. The universal implementation allows us to utilize a "Null Object Pattern" so that there are no error on page renders if the nested object value doesn't exist, but there can be some side effects. I am meeting with Cat tomorrow morning to discuss further. Will give you an update from what comes out of that.